react/src/Groups/index.tsx

```diff
@@ -31,17 +31,16 @@ const useStyles = makeStyles((theme: Theme) => ({
 
 export default function Groups() {
     const classes = useStyles();
-    const [groups, setGroups] = useState<Group[]>([]);
-    const [openNewGroup, setOpenNewGroup] = useState<boolean>(false);
+    const groups = useGroups();
+    const groupPatch = useGroupsPatch();
+    const groupDelete = useGroupsDelete();
+    const [openNewGroup, setOpenNewGroup] = useState(false);
```

Comment:
It seems like on the rerender now when we add or remove a group, `/api/groups` and each group are re-fetched now and I guess this makes sense due to cache invalidation but I don't think this was the behaviour before?

Reply:
Yeah this wasn't the behaviour before. I'm looking into it.